src/main.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import './styles/index.scss'
 import { RouterProvider } from 'react-router-dom'
+import { Spinner } from 'react-bootstrap'
 import AuthProvider from './contexts/auth/AuthProvider.tsx'
 import NotificationArea from './components/NotificationArea.tsx'
 import router from './router.tsx'
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
-    <AuthProvider>
-      <NotificationArea />
-      <RouterProvider router={router} />
-    </AuthProvider>
+    <React.Suspense fallback={<Spinner />}>
```

Comment:
Was this just experimentation or did it tie into this work somehow? I'd lean towards removing it since global loading state seems a bit out of scope for this ticket.

Reply:
Ah yeah I should have mentioned -- It's a recommended implementation to add a `Suspense` boundary. With `i18next-http-backend` that builds the translations  with static files, translations are fetched async. Check out `i18next-react` example code [here](https://github.com/locize/react-i18next-example-app/blob/0b6435faafb9178e793022346a9276de4440d07a/src/App.js#L58-L65).